help/sources/tutorials/ui/create/marketing-automation/pathfactory.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Connect Your PathFactory Account To Experience Platform Through the UI
+description: Learn how to connect your PathFactory account to Experience Platform through the UI.
+exl-id: 
```

Comment:
We can remove exl-id here too and add a line for badge: Beta

Reply:
Fixed